app/src/main/AndroidManifest.xml

```diff
@@ -46,11 +46,14 @@
         <activity
             android:name=".ui.activities.ChangePasswordActivity"
             android:theme="@style/SettingsActivity" />
+        <activity
```

Comment:
According to this comment you left on previous PR https://github.com/kickstarter/android-oss/pull/1243#discussion_r632628450 the activity should be called `RootCommentsActivity`

Reply:
Oh, iOS hasnt changed their naming yet but i think the idea is that they will do it in the future? I can update the classes to be prefixed with "Root" now if we want to get it over with sooner than later.